src/includeDataProps.js

```diff
@@ -0,0 +1,11 @@
+export default function (props) {
+  const dataProps = {}
+
+  Object.keys(props).forEach(key => {
+    if (key.indexOf('data-') > -1) {
```

Comment:
Ah, yes. You're right. It should only match strings that start with `data-`. I'll make this change.